### PR TITLE
Support schema v1 in mpdev install published Marketplace app

### PR DESCRIPTION
**Pull request by huyhg**

### CODE REVIEW

File: docs/mpdev-references.md [hunk 1]
```diff
@@ -162,12 +161,20 @@ APPLICATION_NAMESPACE=default
 # no typos in the names. This should succeed.
 kubectl get application/$APPLICATION_INSTANCE -n $APPLICATION_NAMESPACE
 
+# Use the following for schema v2.
```
Comment (AndreyChoi) [inline docs/mpdev-references.md]: Maybe include how to tell if the application uses schema v1 vs v2?

Comment (huyhg) [reply docs/mpdev-references.md]: Done.

**Pull request merged**